build-coordinator/pom.xml

```diff
@@ -45,6 +45,10 @@
     </dependency>
     <dependency>
       <groupId>org.jboss.pnc</groupId>
+      <artifactId>rest-model</artifactId>
```

Comment:
I was going to ask for this anyway, since we need some method of deserializing responses from PNC in the brew bridge, and it'd be helpful if we didn't have to write our own model classes for this.